templates/override.j2

```diff
@@ -1,3 +0,0 @@
-[Service]
-ExecStart=
-ExecStart=/usr/bin/dockerd -H unix:// {{ docker_opts }}
```

Comment:
I would keep the override... the default got `ExecStart=/usr/bin/dockerd -H fd:// --containerd=/run/containerd/containerd.sock`